test/amd.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit AMD Test Suite</title>
+	<link rel="stylesheet" href="../dist/qunit.css">
+</head>
+<body>
+	<div id="qunit"></div>
+	<script>
+	var beginData,
+		asyncDelay = 1000,
+		times = {
+			pageLoaded: 0,
```

Comment:
These actually only have values because I started from a copy of the autostart test. If I had written this from scratch, this would just be an empty object.